backend/src/main/kotlin/app/ehrenamtskarte/backend/migration/migrations/V0026_AddFreinetAgenciesTable.kt

```diff
@@ -0,0 +1,27 @@
+package app.ehrenamtskarte.backend.migration.migrations
+
+import app.ehrenamtskarte.backend.migration.Migration
+import app.ehrenamtskarte.backend.migration.Statement
+
+/**
+ * Adds freinetagencies table
+ */
+@Suppress("ClassName")
+internal class V0026_AddFreinetAgenciesTable : Migration() {
+    override val migrate: Statement = {
+        exec(
+            """
+                CREATE TABLE freinetagencies (
+                    id SERIAL PRIMARY KEY ,
```

Comment:
💭 The recommended way to declare sequence ID columns for modern PostgreSQL is using
[identity columns](https://www.postgresql.org/docs/17/ddl-identity-columns.html).

Reply:
Yes, an identity column can be (but is not necessarily) a primary key. I don't know if it is a good idea to change the way primary keys are defined mid-project, although I do not see any larger problems with it either. I was just trying to give a hint here.